src/redprl/redprl.grm

```diff
@@ -80,7 +80,8 @@ end
 
 %nonassoc SQUIGGLE
 %right LEFT_ARROW RIGHT_ARROW DOUBLE_PIPE SEMI
-%left FUN_APP AT_SIGN
+%left FUN_APP
+%left AT_SIGN
```

Comment:
Short version: this makes the `@` symbol bind more tightly than function application, which fixes some conflict I ran into at some point. I can dig up more details if you want.

It's not actually clear to me which one should bind more tightly, but I tried this and it seems to work. Best justification for anything :)